src/Lift.ts

```diff
@@ -476,20 +477,16 @@ export class Lift {
       return 'No migration to roll back'
     }
 
-    if (n && n > appliedMigrations.length) {
+    if (!appliedMigrations.length) {
       throw new Error(
-        `You provided ${chalk.redBright(`n = ${chalk.bold(String(n))}`)}, but there are only ${
-          appliedMigrations.length
-        } applied migrations that can be rolled back. Please provide ${chalk.green(
-          String(appliedMigrations.length),
-        )} or lower.`,
+        `Your filter ${chalk.redBright(
+          `n = ${chalk.bold(String(n))}`,
+        )}, does not match any number of applied migrations that can be rolled back. Please provide a valid <dec|name|timestamp>.`,
```

Comment:
I think you mean `increment` here. Also, it is not very clear by `inc` what the value actually does so would recommend using the full doesn't imply what it actually does so use the full form

Reply:
> I think you mean increment instead of dec here.

good catch 😉 

> Also, it is not very clear by inc what the value actually does

agreed 👍 

> would recommend using the full doesn't imply what it actually does so use the full form

can you expand on that? I'm not sure I follow 😕

Reply:
It was not very clear what `dec` meant. I can interpret that as `decimal` instead of decrement.

So I would recommend you to change it to `<decrement|name|timestamp>`

Reply:
This is what we are currently using in the CLI @pantharshit00 - Sergio just wrote the code to so the CLI _actually_ supports this functionality ;) So if we want to change this naming, we should create a new issue about it. It should not be part of this PR.

Reply:
Sorry, I wasn't aware of that. I will create a new issue after this is merged.